### PR TITLE
pass location to map

### PR DESCRIPTION
**Pull request by jocelynshen**

### CODE REVIEW

**Review by clarisaleu — APPROVED** [review]

File: app/src/main/java/com/example/footprnt/Discover/Adapters/ListAdapter.java [hunk 1]
```diff
@@ -80,8 +84,11 @@ public int getItemCount() {
         TextView tvBusinessAddress;
         TextView tvBusinessPhone;
         ImageView btnCall;
+        ImageView btnLocation;
```
Comment (ajankenm) [inline app/src/main/java/com/example/footprnt/Discover/Adapters/ListAdapter.java:87]: mVariableName format please

File: app/src/main/java/com/example/footprnt/Discover/Adapters/ListAdapter.java [hunk 2]
```diff
@@ -130,10 +140,38 @@ public void bindBusinessDetail(Business business) {
             } catch (Exception e) {
                 rating.setVisibility(View.INVISIBLE);
             }
+            final String imageUrl = business.getImageUrl();
+            if (imageUrl == null || imageUrl.length() == 0){
```
Comment (ajankenm) [inline app/src/main/java/com/example/footprnt/Discover/Adapters/ListAdapter.java:144]: TextUtils.isEmpty

File: app/src/main/java/com/example/footprnt/Discover/Adapters/ListAdapter.java [hunk 3]
```diff
@@ -130,10 +140,38 @@ public void bindBusinessDetail(Business business) {
...
+                Picasso.with(mContext).load("https://pyzikscott.files.wordpress.com/2016/03/yelp-placeholder.png?w=584");
```
Comment (ajankenm) [inline app/src/main/java/com/example/footprnt/Discover/Adapters/ListAdapter.java:145]: Static URL should be a constant so it's easily findable and can be changed if needed

File: app/src/main/java/com/example/footprnt/Map/MapFragment.java [hunk 4]
```diff
@@ -818,4 +819,24 @@ private void toggleMenuItem(MenuItem menuItem, int id) {
...
         }
     }
+
+    /**
+     * Zooms in on business location and creates marker at the business location
+     * @param address address of business
+     * @param businessName name of business
+     * @param imageUrl url of business image
+     */
+    public void handleDiscoverInteraction(String address, String businessName, String imageUrl){
```
Comment (ajankenm) [inline app/src/main/java/com/example/footprnt/Map/MapFragment.java:829]: Mark imageUrl with @Nullable if you're potentially expecting a null

**Pull request merged**